funnel/templates/index.html.jinja2

```diff
@@ -226,8 +226,9 @@
                 </a>
                 <div class="card__body card__body--cfp">
                   <h3 class="card--cfp__title mui--text-title zero-top-margin zero-bottom-margin"><a href="{{ project.url_for() }}" class="js-truncate" data-truncate-lines="2">{{ project.title }}</a></h3>
-                  {% if project.cfp_end_at %}<p class="mui--text-caption zero-top-margin mui--text-danger">Last date of submission - {{ project.cfp_end_at|longdate }}</p>{% endif %}
                   <p class="mui--text-subhead mui--text-light zero-top-margin card--cfp__tagline js-truncate-readmore" data-truncate-lines="2">{{ project.tagline }}</p>
+                  <hr class="separator">
+                  {% if project.cfp_end_at %}<p class="mui--text-body1 zero-top-margin"><em><span class="mui--text-light">Submission closes on </span><span class="secondary-color-txt ">{{ project.cfp_end_at|longdate }}</span></em></p>{% endif %}
```

Comment:
Needs `{% trans %}` tag.

Reply:
The `longdate` filter will return time in the user's timezone. There's a decision to be made here on whether this should be project or user timezone.

Reply:
Added cfp_end_at_localized property to get time in project timezone